cluster/aws/wily/util.sh

```diff
@@ -33,6 +33,10 @@ function detect-wily-image () {
         AWS_IMAGE=ami-3355505d
         ;;
 
+      ap-northeast-2)
+        AWS_IMAGE=ami-e427e98a
```

Comment:
For the record, I verified this ID against https://cloud-images.ubuntu.com/locator/